### PR TITLE
Issue 34 phantom js pool fully used

### PR DESCRIPTION
**Pull request by Dwightojo**
This fixes the PhantomJSPool from filling up because of windows access errors. Now whenever such an error is encountered the entire pool is closed.

###### Author:

- [x] No IntelliJ warnings on new code (including spelling)
- [x] No bugs/vulnerabilities on [sonar](https://sonarcloud.io/dashboard?branch=Issue-34-PhantomJS-pool-fully-used&id=nl.xillio.xill%3Axill-build)
- [x] High line coverage on new code (any higher would need a rewrite)
- [x] Tests cover all (reasonable) edge cases
- [x] Reformat code doesn't leave any file modified

###### Reviewers:

- [X] All _Creator_ checkboxes are rightfully checked
- [ ] The code is clearly structured (no large methods, clear naming, good separation of concerns, etc.)
- [ ] The code has enough helpful documentation and comments
- [ ] The implementation conforms to the issue description
- [ ] [Do's and don'ts](https://help.xillio.com/conventions/java-code/) are observed

### CODE REVIEW

File: plugin-web/src/main/java/nl/xillio/xill/plugins/web/services/web/WebServiceImpl.java [hunk 1]
```diff
@@ -355,7 +361,15 @@ public void quit(final WebVariable var) {
 
     @Override
     public PhantomJSPool.Entity getEntityFromPool(final PhantomJSPool pool, final Options options) {
-        PhantomJSPool.Entity pjsInstance = pool.get(pool.createIdentifier(options), this);
+        PhantomJSPool.Entity pjsInstance = null;
```
Comment (sanvisser) [inline plugin-web/src/main/java/nl/xillio/xill/plugins/web/services/web/WebServiceImpl.java]: This is in the Java conventions as an anti-pattern:

https://help.xillio.com/conventions/java-code/#limit-scope-as-much-as-possible

Comment (zhochmann) [reply plugin-web/src/main/java/nl/xillio/xill/plugins/web/services/web/WebServiceImpl.java]: I am just curious - is mentioned case considered to be anti-pattern only because someone in Xillio does not like it or is there any solid explanation from some Java expert?

Comment (sanvisser) [reply plugin-web/src/main/java/nl/xillio/xill/plugins/web/services/web/WebServiceImpl.java]: I don't exactly know however I think the explanation given is valid:
`The scope of local variables can often be limited to increase readability, testability and general maintainability of the code. `

So I think in general it is good to do it.
